feedback.md

```diff
@@ -7,17 +7,20 @@
 |   Answered comprehension questions 	|   	|
 |   Both Teammates contributed to the codebase	|   	|
 |   Regular Commits with meaningful commit messages 	|   	|
-|  **Wave 1**	|   	|
-|  `Scoring` class	|   	|
+|  **Scoring class**	|   	|
 |  `score` method	|   	|
-|   Uses hash to store the alpha score	|   	|
+|   Uses appropriate data structure to store the alpha score	|   	|
 |   Tests `score` edge cases (one-letter word, 7 letter word)	|   	|
 |  `highest_scoring_word` method	|   	|
-|   Appropriately handling edge cases	|   	|
+|   Appropriately handles edge cases	|   	|
 |   Tests `highest_scoring_word` edge cases (ties, no words, no array)	|   	|
-|  **Wave 2**	|   	|
-|  `Player` class	|   	|
-|	**Baseline**	|	|
-|  Readable code with consistent indentation. 	|   	|
-|   **Extras**	|   	|
-
+|  **Player class**	|   	|
+|  Uses an array to keep track of words played	|   	|
+|  Uses existing code to create `highest_scoring_word` and `highest_word_score`	|   	|
+|  Returns `true` or `false` from `won?` method |   |
+|  Tests edge cases on the `play(word)` method for words that cannot be scored |     |
+|  **TileBag class**	|   	|
+|  Uses a data structure to keep the set of default tiles in the bag	|   	|
```

Comment:
They would always have to store the _active_ tiles in a bag using an array or hash, because that will change over time as the application runs. The _default_ tiles could, in theory, be stored w/o an array however:

```ruby
class TileBag
  def initialize
    @tiles = []
    9.times { @tiles << "A" }
    2.times { @tiles << "B" }
    #...
    @tiles << "Z"
  end
end
```

I'm not sure if we care about whether they use this approach to initializing the active tile collection or not.